### PR TITLE
Humble release

### PR DESCRIPTION
**Pull request by Acuadros95**

### CODE REVIEW

**Comment by pablogs9** [comment]
CI not launched because of this error:

image

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -3,36 +3,35 @@ name: CI micro-ROS Agent
...
-            cp -R urosagent/* /uros_ws/src/uros/micro-ROS-Agent/
-
-        - name: Build
-          run:  |
-            cd /uros_ws
-            . /opt/ros/$ROS_DISTRO/setup.sh
-            . install/local_setup.sh
-            ros2 run micro_ros_setup build_agent.sh
\ No newline at end of file
+      runs-on: ${{ matrix.os }}
```
Comment (Acuadros95) [inline .github/workflows/ci.yml:13]:
```suggestion
      runs-on: ubuntu-20.04
```

**Comment by pablogs9** [comment]
Could you prepare the backports for foxy and galactic of this CI approach @Acuadros95 ?

**Review by pablogs9 — APPROVED** [review]

**Comment by Acuadros95** [comment]
@mergify backport foxy galactic

**Pull request merged**